packages/flutter/lib/src/gestures/monodrag.dart

```diff
@@ -89,7 +88,7 @@ abstract class DragGestureRecognizer extends OneSequenceGestureRecognizer {
   /// at (500.0, 500.0). If it is instead set to [DragStartBehavior.start],
   /// [onStart] will be called at the time corresponding to the touch's position
   /// at (510.0, 500.0).
-  DragStartBehavior dragStartBehavior;
+  DragStartBehavior? dragStartBehavior;
```

Comment:
This one is surprising. I expected this to be non-nullable.

Reply:
Not sure why I made it nullable :-/ Changed